components/home/Challenge.tsx

```diff
@@ -37,18 +39,22 @@ export function Challenge({
     return <ThemedText>Loading Challenge...</ThemedText>;
   } else {
     return (
-      <ThemedView style={{ backgroundColor: "transparent" }}>
+      <ThemedView 
+        key={index}
+        testID={testID} 
+        style={{ backgroundColor: "transparent" }}
+      >
         <TouchableOpacity
           onPress={() => setIsOpen(!isOpen)}
           activeOpacity={0.8}
         >
           <ThemedView style={[styles.challenge]}>
             {/* 
               Challenge Image 
-              source={{uri: challengeData.image_id}}
+              source={require("@/assets/images/challenge2.png")}
```

Comment:
Why was this comment updated ? I'm not sure to understand if that comment is important.